src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/condition-recommendations.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Get condition recommendations
+metaDescription: For APM entities, you can see and create recommended alert conditions.
+watermark: Limited release
+---
+
+Use the New Relic Navigator's mini-overview to get alert condition recommendations for your monitored entities. With these recommendations, you can create alerts conditions to be notified when your entities aren't performing as they should.
+
+<Callout variant="caution">
+Condition recommendations are only available for APM-monitored entities.
+</Callout>
+
+## Why it matters
+
+Condition recommendations are all about making it easier for you to find entities and their conditions and thresholds you'd want to be notified about.
+
+Every recommendation includes one or more metrics that might interest you, as well as a threshold beyond which you might want to be notified.
+
+## Get condition recommendations
+
+From the entity detail pane, you can quickly create an alert condition. When you click **Add recommended conditions**, the UI provides a list of recommended conditions you might want to be notified about.
```

Comment:
This section almost seems like it could be a short procedure. You might include something about checking/changing the default thresholds.

Reply:
Yeah, I sort of wanted to stay away from a procedure. This UI changes so much all the time...